eng/common/TestResources/New-TestResources.ps1

```diff
@@ -154,14 +154,21 @@ if ($ProvisionerApplicationId) {
     $subscriptionArgs = if ($SubscriptionId) {
```

Comment:
Any changes under eng/common will be overwritten if they aren't made in the azure-sdk-tools repo as well, see https://github.com/Azure/azure-sdk-for-net/tree/master/eng/common#updating

Reply:
Thanks for the heads-up.  I was not aware.

Reply:
These changes have been moved to the appropriate location in the tools repo.